src/adapters/postgres/user-adapter.ts

```
@@ -182,18 +182,27 @@ export class PostgresUserService implements IServicelocator {
       const requestBody = request?.body || {};
       const roleFromBody = requestBody.role;
 
+      // Admin roles that use the admin account verification template
+      const adminRoles = [
+        'Regional Admin',
```

ESLint

[error] Replace `'Regional·Admin'` with `"Regional·Admin"`

(prettier/prettier)